static/styles/zulip.scss

```diff
@@ -2224,18 +2207,37 @@ div.floating_recipient {
     }
 }
 
-div.topic_edit_spinner {
-    display: inline-block;
-    width: 18px;
-    height: 18px;
-    margin-top: -1px;
-    padding: 2px;
-    vertical-align: middle;
-}
+div {
```

Comment:
I think this commit is probably counterproductive, so I skipped it.  These don't really represent a common set of related CSS we want to organize, but instead a set of N unrelated things that happen to all involve divs.